options.go

```diff
@@ -13,12 +13,23 @@ type Options struct {
 	// Print step definitions found and exit
 	ShowStepDefinitions bool
 
-	// Run scenarios in random order.
+	// Randomize causes scenarios to be run in random order.
 	//
-	// This is especially helpful for detecting situations
-	// where you have state leaking between scenarios, which
-	// can cause flickering or fragile tests.
-	RandomOrder bool
+	// Randomizing scenario order is especially helpful for detecting
+	// situations where you have state leaking between scenarios, which can
+	// cause flickering or fragile tests.
+	Randomize bool
```

Comment:
This is actually what I had in the first revised commit 0eed963 if you want to take a look. But you do need 3 categories of preference here:

1. do not randomize (most common case, with default nil value `0`),
2. randomize and let godog pick the seed (secondary common case),
3. randomize with a specific seed to try to reproduce a random failure (rare case). 

For the secondary case, I was originally solving in 0eed963 by having a magic value `-1` put as the seed.  This felt fine when I was interacting via Options struct.

However, once I later started mapping the Options struct to the command UI flags it became very confusing: passing something like `--random-seed=-1` for the more common randomization scenario was strange user interface.  Once I moved to the two variable model to have a more consistent mapping between the CLI and the Options struct, and it felt more understandable to work with.

It would also be possible to do something like this _just_ in the CLI and transform it based on rules before piping it back into the Options struct, but it appears that would require re-architecting the way Options/Flags interact currently.

For comparison, the way the CLI options on Ruby Cucumber CLI work are:
```
 --order random          (random)
 --order random:12345    (random with specified seed)
```

Let me know what you think!